runtime/src/main/scala/akka/grpc/internal/Identity.scala

```diff
@@ -1,8 +1,4 @@
-/*
- * Copyright (C) 2018-2020 Lightbend Inc. <https://www.lightbend.com>
- */
-
-package akka.grpc
+package akka.grpc.internal
```

Comment:
Copyright now missing? Also in other moved files.

Do we put `@InternalApi` on classes in an `internal` package? For me, the package name seems to be a strong enough indication.